keanu-project/src/main/proto/io/improbable/keanu/BayesianNetwork.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package keanu;
+
+option java_package = "io.improbable.keanu";
+option java_outer_classname = "KeanuSavedBayesNet";
+
+message BayesNet {
+    repeated Vertex vertices = 1;
+}
+
+message Vertex {
+    uint32 id = 1;
+    string label = 2;
+    string vertexType = 3;
+    VertexValue value = 4;
+    repeated HyperParams hyperParameters = 5;
+}
+
+message VertexValue {
+    enum ValueType {
+        DOUBLE = 0;
+        INT = 1;
+        BOOL = 2;
+        STRING = 3;
```

Comment:
Do we need a `GENERIC` ValueType?

Reply:
Yep - good call - I've added support for generic via a byte array.